tools/compare-requests.ts

```diff
@@ -34,6 +34,7 @@ const extension = {
   python: "py",
   r: "r",
   // rust: 'rs',
+  http: "http",
```

Comment:
You shouldn't enable this by adding it here if you don't add code for running the `curl` command and comparing the resulting HTTP
```suggestion
  // http: "txt",
```